nxdrive/client/remote_client.py

```diff
@@ -815,8 +816,15 @@ def get_fs_children(
         self, fs_item_id: str, filtered: bool = True
     ) -> List[RemoteFileInfo]:
         children = self.execute(command="NuxeoDrive.GetChildren", id=fs_item_id)
-        infos = [RemoteFileInfo.from_dict(fs_item) for fs_item in children]
-
+        infos = []
+        for fs_item in children:
+            try:
+                infos.append(RemoteFileInfo.from_dict(fs_item))
+            except UnknownDigest:
+                log.debug(
+                    f"Ignoring unsyncable document {fs_item!r} because of unknown digest {fs_item['digest']}"
```

Comment:
As the digest is part of the dict, it will be printed by default.
```suggestion
                    f"Ignoring unsyncable document {fs_item!r} because of unknown digest"
```